tests/btrfs-progs/install.pm

```diff
@@ -18,6 +18,7 @@ use utils;
 use testapi;
 
 my $STATUS_LOG = '/opt/status.log';
```

Comment:
I would rather use constant here instead of var. Also, capital letters suggest me of a constant :)

Reply:
Use constant need modify all sentences which invoke these. And bare-word not allowed while "strict subs", "Readonly" maybe can resolve it but it's not a core module.
How about I change var name to lowercase?